biogeophys/FatesHydroWTFMod.F90

```diff
@@ -28,7 +28,9 @@ module FatesHydroWTFMod
        __FILE__
 
 
-  real(r8), parameter :: min_ftc = 0.00001_r8   ! Minimum allowed fraction of total conductance
+!  real(r8), parameter :: min_ftc = 0.00001_r8   ! Minimum allowed fraction of total conductance
+!  The above cause negative organ water content
+  real(r8), parameter :: min_ftc = 0.00001e2_r8   ! Minimum allowed fraction of total conductance
```

Comment:
@pnlfang is this something you want to keep? If so, can you make a note here in the comments that briefly explains the intent and that it was changed?

Reply:
No.